project.clj

```diff
@@ -14,28 +10,24 @@
 (defproject puppetlabs/puppetserver ps-version
   :description "Puppet Server"
 
-  :dependencies [[org.clojure/clojure ~clj-version]
-
-                 ;; begin version conflict resolution dependencies
-                 [puppetlabs/typesafe-config "0.1.5"]
-                 [org.clojure/tools.macro "0.1.5"]
```

Comment:
fyi, we're getting an ![older version](https://github.com/puppetlabs/clj-parent/blob/0.1.3/project.clj#L17) from `clj-parent` currently  - 0.1.2.